docs/42-faq.md

```diff
@@ -0,0 +1,20 @@
+# FAQ
+
+## Is doAzureParallel available on CRAN?
+No. At the moment doAzureParallel is only being distributed via GitHub.
+
+## Which version of R does doAzureParallel use?
+By default, doAzureParallel uses Microsoft R Open 3.3.
+
+## Does doAzureParallel support a custom version of R?
+No. We are looking into support for different versions of R as well as custom version of R but that is not supported today.
+
+## How much does doAzureParallel cost?
```

Comment:
I would also just be super explicit and obvious and state "doAzureParallel itself is free to use" .. just because i've gotten this question from people before.